packages/react-core/src/demos/DescriptionList/examples/DescriptionListDrawer.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import {
+  Gallery,
+  PageSection,
+  GalleryItem,
+  Card,
+  CardBody,
+  Drawer,
+  DrawerPanelContent,
+  DrawerContent,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  PageSectionVariants,
+  TextContent,
+  Text,
+  DrawerPanelBody,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Button,
+  Page,
+  Title
+} from '@patternfly/react-core';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+export const DescriptionListDrawer: React.FunctionComponent = () => {
+  const drawerRef = React.useRef<HTMLDivElement>();
+  const [isExpanded, setIsExpanded] = React.useState(true);
+
+  const onExpand = () => {
+    drawerRef.current && drawerRef.current.focus();
+  };
+
+  const onCloseClick = () => {
+    setIsExpanded(false);
+    drawerRef.current && drawerRef.current.focus();
+  };
+
+  const onOpenDrawer = () => {
+    setIsExpanded(true);
+  };
+
+  const panelContent = (
+    <DrawerPanelContent>
+      <DrawerHead>
+        <Title headingLevel="h2">test</Title>
+        <DrawerActions>
+          <DrawerCloseButton onClick={onCloseClick} />
+        </DrawerActions>
+      </DrawerHead>
+      <DrawerPanelBody>
+        <DescriptionList isFillColumns columnModifier={{ default: '2Col', lg: '2Col' }}>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Name</DescriptionListTerm>
+            <DescriptionListDescription>test</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Namespace</DescriptionListTerm>
+            <DescriptionListDescription>
+              <a href="#">mary-test</a>
+            </DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Labels</DescriptionListTerm>
+            <DescriptionListDescription>app=test</DescriptionListDescription>
+          </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Pod selector</DescriptionListTerm>
+            <DescriptionListDescription>Nod selector is not available at this time</DescriptionListDescription>
```

Comment:
Nit but looks like this is misspelled (core demo should be updated, too)

```suggestion
            <DescriptionListDescription>Node selector is not available at this time</DescriptionListDescription>
```